retail-managment-system/rms-ui/src/app/auth/components/login/login.component.scss

```diff
@@ -0,0 +1,39 @@
+mat-card {
+    max-width: 600px;
+    margin: 5em auto;
+    text-align: center;
+    max-height: 600px;
+  }
+  .main {
+    height: 630px;
+  }
+  
+  .header{
+    text-align: center;
+    font-weight: bold;
+    font-size: large;
```

Comment:
text-align: center;
    font-weight: bold;
    font-size: large;
   --->  are this exist in bootstrap or not?
    and any thing like this in file css